src/read.c

```diff
@@ -1185,60 +1185,6 @@ static ArgList ReadFuncArgList(ScannerState * s,
 }
 
 
-void StartFakeFuncExpr(Int startLine)
-{
-    assert(STATE(IntrCoding) == 0);
-
-    // switch to coding mode now
-    CodeBegin();
-
-    // code a function expression (with no arguments and locals)
-    Obj nams = NEW_PLIST(T_PLIST, 0);
-
-    // If we are in the break loop, then a local variable context may well
-    // exist, and we have to create an empty local variable names list to
-    // match the function expression that we are creating.
-    //
-    // Without this, access to variables defined in the existing local
-    // variable context will be coded as LVAR accesses; but when we then
-    // execute this code, they will not actually be available in the current
-    // context, but rather one level up, i.e., they really should have been
-    // coded as HVARs.
-    //
-    // If we are not in a break loop, then this would be a waste of time and
-    // effort
-    if (LEN_PLIST(ReaderState()->StackNams) > 0) {
-        PushPlist(ReaderState()->StackNams, nams);
```

Comment:
The only change made to the two functions when copying them was to replace `ReaderState()->StackNams` by an explicit argument to `StartFakeFuncExpr` resp. `FinishAndCallFakeFuncExpr`